### PR TITLE
[angelscript] Update to 2.35.1

### PR DESCRIPTION
**Pull request by crackedmind**
**Describe the pull request**

- #### What does your PR fix?  
  Update port to latest version 2.35.1 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all / no

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  `Yes`

### CODE REVIEW

File: ports/angelscript/vcpkg.json [hunk 1]
```diff
@@ -1,9 +1,20 @@
 {
   "name": "angelscript",
-  "version-string": "2.35.0",
+  "version-string": "2.35.1",
+  "port-version": 1,
```
Comment (JonLiu1993) [inline ports/angelscript/vcpkg.json:4]:
```suggestion
  "version": "2.35.1",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by crackedmind** [comment]
@JonLiu1993 lol, want i need to do to fix this ? :D

**Comment by JonLiu1993** [comment]
> @JonLiu1993 lol, what i need to do to fix this ? :D

Please run command "./vcpkg x-add-version angelscript --overwrite-version" and commmit again

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:

- x86-windows
- x64-windows
- x64-windows-static

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the version update! I pushed some minor whitespace cleanup and will merge this when the build comes back.

**Pull request merged**